x-pack/plugins/spaces/public/nav_control/nav_control_popover.tsx

```diff
@@ -179,7 +179,7 @@ class NavControlPopoverUI extends Component<Props, State> {
   }
 
   private getAlignedLoadingSpinner() {
-    return <EuiLoadingSpinner size="m" className="eui-alignMiddle" />;
+    return <EuiSkeletonRectangle borderRadius="m" contentAriaLabel="Loading spaces navigation" />;
```

Comment:
Does this `contentAriaLabel` need to be translated?

Reply:
ideally yes